package.json

```diff
@@ -17,9 +21,9 @@
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
+    "start": "craco start",
+    "build": "craco build",
+    "test": "craco test",
```

Comment:
@craco/craco setup.

Reply:
This seems odd…we are replacing our app scripts with craco stuff?

Reply:
Not sure why we need to do it. I just follow the documentations. I will do some research about it.

Reply:
OK, thanks.

Reply:
From [Cesiumjs documentation](https://cesium.com/learn/cesiumjs-learn/cesiumjs-quickstart/), CesiumJS requires a few static files to be hosted on our server and we also need to set a global variable before we use Cesium. However, we want to use Ceisumjs as a dependency in CRA. [craco-cesium](https://github.com/reearth/craco-cesium) is a  [CRACO](https://github.com/gsoft-inc/craco) plugin to configure cesium into our module bundler. Based on the design of CRA, we don't have access to the WebPack configuration but we do want to override our module bundler. Therefore, we need to use [CRACO](https://github.com/gsoft-inc/craco), one of the methods to override CRA WebPack configuration. To use `CRACO`, we need to create a new `craco.config.js` file to include additional plugins that need to be added to our project. By changing our `scripts` command from `react-scripts` to `craco`, CRACO will customize our app WebPack Configure based on the original CRA configuration and additional setup. Help it helps!

Reply:
Ok, I will do it today!